docs/source/user_guide/quickstart.rst

```diff
@@ -253,29 +253,31 @@ An example output looks like this:
    (my_env) user@hostname:/path/to/quickstart$ cognify optimize workflow.py 
    > light_opt_layer | (best score: 0.16, lowest cost@1000: 0.09 $): 100%|███████████████| 10/10 [01:53<00:00, 11.30s/it]
    ================ Optimization Results =================
-   Num Pareto Frontier: 2
+   Number of Optimization Results: 2
    --------------------------------------------------------
-   Pareto_1
+   Optimization_1
+   Quality improves by 5%
+   Cost is 0.95x original
    Quality: 0.160, Cost per 1K invocation: $0.28
-   Applied at: light_opt_layer_4
    --------------------------------------------------------
-   Pareto_2
+   Optimization_2
+   Quality improves by 7%
+   Cost is 0.98x original
    Quality: 0.154, Cost per 1K invocation: $0.09
-   Applied at: light_opt_layer_6
    ========================================================
 
 The optimizer found two optimized workflow versions on the Pareto frontier, i.e., they are the most cost-effective solutions within all searched optimizations.
 
 .. note::
 
-   It's not guaranteed that the optimizer will find any better solutions than the original one. You might get ``Num Pareto Frontier: 0`` in the output.
+   It's not guaranteed that the optimizer will find any better solutions than the original one. You might get ``Number of Optimization Results: 0`` in the output.
```

Comment:
shouldn't we be only display this message if the number of frontier is 0?

Reply:
So should it not be in the documentation at all?

Reply:
oh.. i somehow didn't see that this is in documentation. then it's fine.